Lib/defconQt/featureTextEditor.py

```diff
@@ -29,6 +29,7 @@ def __init__(self, font=None, parent=None):
         self.editor.modificationChanged.connect(self.setWindowModified)
 
     def setWindowTitle(self, title="Font Features", font=None):
+        title = self.tr("Font Features")
```

Comment:
This should be a single string, not split into translatable and non-translatable parts.